hook-tests/600-no-debian.test

```diff
@@ -265,6 +263,6 @@ else
   exit 1
 fi
 
-# cleanup temp files used for comparisons 
+# cleanup temp files used foe comparisons 
```

Comment:
This looks like a typo

Reply:
Indeed, that's corrected now with the latest commit.